lib/cast.rb

```diff
@@ -0,0 +1,20 @@
+class Cast
+  def self.to_bool(value)
+    case value.class
+
+      when String
+        return true if self == true || self =~ (/(true|t|yes|y|1)$/i)
```

Comment:
You use `self` everywhere but I think you mean `value`. Also, it's weird that this is checking if value is true if we know it's a String.